lib/authoritex/loc/base.ex

```diff
@@ -102,6 +81,39 @@ defmodule Authoritex.LOC.Base do
         end
       end
 
+      defp parse_fetch_result(%{body: response, status_code: 200}) do
+        with doc <- SweetXml.parse(response) do
+          case doc |> SweetXml.xpath(~x"/rdf:RDF") do
+            nil ->
+              {:error, {:bad_response, response}}
+
+            rdf ->
+              {:ok,
+               SweetXml.xpath(rdf, ~x"./madsrdf:*",
+                 id: ~x"./@rdf:about"s,
+                 label: ~x"./madsrdf:authoritativeLabel[1]/text()"s,
+                 qualified_label: ~x"./madsrdf:authoritativeLabel[1]/text()"s,
+                 hint: ~x"./no_hint/text()",
+                 variants:
+                   ~x".//madsrdf:variantLabel/text()"ls
+                   |> SweetXml.transform_by(&Enum.uniq/1)
```

Comment:
More of a curiosity question than anything else, did you notice that there were duplicate variants in LOC records or is this just to make sure? It's definitely not a huge performance issue because I've never seen more than a handful of variants on a record in any case.

Reply:
I did not but @mbklein suggested I add that. See above ^

Reply:
@bmquinn From my comment above:

```
iex(2)> Authoritex.fetch("http://id.loc.gov/authorities/subjects/sh2010008400")
{:ok,
%{
  hint: nil,
  id: "http://id.loc.gov/authorities/subjects/sh2010008400",
  label: "Cooking",
  qualified_label: "Cooking",
  variants: ['Cookery', 'Cuisine', 'Food preparation', 'Cookery', 'Cookery']
}}
```

Reply:
Right on, sounds good!